docs/release-notes/1.1.1.asciidoc

```diff
@@ -4,6 +4,14 @@
 [[release-notes-1.1.1]]
 == {n} version 1.1.1
 
+[[breaking-1.1.1]]
+[float]
+=== Breaking changes
+
+* Revert readiness probes independent of the host/container... (#2528) {pull}3060[#3060]
+
+
+
```

Comment:
Should we manually remove some empty lines here?

Reply:
When I looked at it in the past it didn't change the rendering at all so I didn't even open an issue 🤷‍♀️